hivemind/utils/__init__.py

```diff
@@ -7,5 +7,5 @@
 from hivemind.utils.networking import *
 from hivemind.utils.serializer import *
 from hivemind.utils.tensor_descr import *
-from hivemind.utils.threading import *
 from hivemind.utils.timed_storage import *
+from hivemind.utils.limits import *
```

Comment:
nit: Please sort them (so it's easy to check whether a particular module is included)